src/main/java/eu/mihosoft/vrl/v3d/TextExtrude.java

```diff
@@ -0,0 +1,165 @@
+package eu.mihosoft.vrl.v3d;
```

Comment:
This is a static class for turning a font and text into a list of CSG's representing one CSG per letter. This will detect and remove holes from the given shapes as well. THe list can be flattened into a single CSG using a union, but if you are just going to perform a difference, it is more efficient to leave it as the list and difference the list.